mappings/net/minecraft/util/math/noise/NoiseSampler.mapping

```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_586 net/minecraft/util/math/noise/SimplexNoiseSampler
+CLASS net/minecraft/class_586 net/minecraft/util/math/noise/NoiseSampler
```

Comment:
Why the rename?

Reply:
It doesn’t generate simplex noise. It’s used to generate specific kinds of noise